moninag/registration/urls.py

```diff
@@ -1,20 +1,20 @@
 from django.conf.urls import url
-from registration.views import activate, inactive_account, login, auth_view, logout, profile, invalid_login, register_user, register_success
+from registration.views import activate, inactive_account, login, auth, logout, profile, invalid_login, register_user, register_success
 
 # Uncomment the next two lines to enable the admin:
 # from django.contrib import admin
 # admin.autodiscover()
 
 urlpatterns = [
     # auth
-    url(r'^$', login, name='login'),
-    url(r'^auth/$', auth_view, name='auth_view'),
+    url(r'^$', auth, name='auth'),
```

Comment:
updat url_path to
auth/logon
auth/register
auth/activate
auth/logout
api/1/profile
api/1/profile/inactive

add functionality auth/recovery 
remove register_success